lib/game.rb

```diff
@@ -25,13 +25,13 @@ def set_up_game
     @computer_board = Board.new(board_size)
 
     @computer_player = ComputerBrain.new(@user_board, @computer_board)
-    @computer_player.cpu_place_ships(@computer_board.get_ships)
+    @computer_player.cpu_place_ships(@computer_board.ships)
```

Comment:
I did the same thing I think. Or maybe had the same thought. We should remember to take a second look after merges.